woocommerce/woocommerce-admin
# add npm run test:zip command

This PR adds `npm run test:zip` to create a test zip in the root directory for easy uploading to a test install. The zip command filters out all the files in `.distignore` and dev packages in the `vendor` folder.

### Detailed test instructions:

- `npm run test:zip`
- Upload to a test site & activate
- Check that the screens work correctly
